src/Domain/System/ActionGateway.php

```diff
@@ -127,6 +127,14 @@ public function selectModuleActionsByRole(
 
         return $this->db()->select($sql, $data);
     }
+
+    public function selectActionByModuleAndRole($gibbonModuleID, $gibbonRoleIDCurrent)
+    {
+        $data = ['gibbonModuleID' => $gibbonModuleID, 'actionName' => 'View CAS in Student Profile', 'gibbonRoleID' => $gibbonRoleIDCurrent];
```

Comment:
It looks like this value may have been accidentally hard-coded. Is it possible to use the isActionAccessible or getHighestGroupedAction functions, rather than creating a new one?